pkg/api/v1/types.go

```diff
@@ -723,6 +723,12 @@ type ClusterSpec struct {
 	// If active the PodNodeSelector admission plugin is configured at the apiserver
 	UsePodNodeSelectorAdmissionPlugin bool `json:"usePodNodeSelectorAdmissionPlugin,omitempty"`
 
+	// EnableUserSSHKeyAgent control whether the UserSSHKeyAgent will be deployed in the user cluster or not.
+	// If it was enabled, the agent will be deployed and used to sync the user ssh keys, that the user attach
+	// to the created cluster. Otherwise, there will be user ssh keys sync(except the one that's added during cluster
+	// creation will be added) in the cluster. The agent is enabled by default
```

Comment:
> Otherwise, there will be user ssh keys sync (except the one that's added during cluster
> creation will be added) in the cluster.

What does this mean?

Reply:
by sync:
> the agent will be deployed and used to sync the user ssh keys, that the user attach to the created cluster.
as mentioned in the top 

If this wasn't clear enough, I can adjust it :)

Reply:
To me it reads "If you enable this, we deploy the agent and sync the keys. Otherwise, there will be user key sync [the exact same thing?!].", so yes, please clarify the wording :-)